avocado/plugins/run.py

```diff
@@ -128,10 +128,13 @@ def configure(self, parser):
 
         if multiplexer.MULTIPLEX_CAPABLE:
             mux = parser.add_argument_group('multiplexer use on test execution')
-            mux.add_argument('-m', '--multiplex-files', nargs='*',
+            mux.add_argument('-m', '--multiplex', nargs='*', dest='multiplex_files',
                              default=None, metavar='FILE',
                              help='Location of one or more Avocado multiplex (.yaml) '
                              'FILE(s) (order dependent)')
+            mux.add_argument('--multiplex-files', nargs='*',
+                             default=None, metavar='FILE',
+                             help='DEPRECATED: please use --multiplex instead')
```

Comment:
Well I think it's more important to add the warning, because users will not notify it unless you put it in front of their eyes. We can create generic `obsolete` function, which can stay in the framework for such cases.